src/site/content/en/progressive-web-apps/pwa-checklist/index.md

```diff
@@ -0,0 +1,482 @@
+helps to bridge the native-web app gap and makes your experience feel more
+native.
+
+#### How
+
+During a service worker's `install` event, you can precache a custom offline
+page for later use. If a user then goes offline and your service worker is
+unable to fetch an HTML page, you can respond with precached custom offline
+page. You can follow our custom
+[offline page sample](https://googlechrome.github.io/samples/service-worker/custom-offline-page/)
+to see a full example  of this in action and learn how to implement it yourself.
+
+{% endDetails %}
+
+
+{% Details %}
+{% DetailsSummary 'h3' %}
+
+Is installable
+
+Users who install or add apps to their home screens tend to engage with those
+apps more.
+
+{% endDetailsSummary %}
+
+Users who install or add apps to their home screens tend to engage with those
+apps more.
+
+#### Why
+
+Installing a Progressive Web App allows it to look, feel, and behave like all
+other installed apps. It's launched from the same place users launch their
+other apps. It runs in its own app window, separate from the browser, and it
+appears in the task list, just like other apps.
+
+Why would you want a user to install your PWA? The same reason you'd want a
+user to install your app from any app store. Users who install your apps
+are your most engaged audience, and have better engagement metrics than
+typical visitors, often at parity with native app users on mobile devices.
+These metrics include more repeat visits, longer times on your site and
+higher conversion rates.
+
+#### How
+
+You can follow our [installable guide](/installable/) to learn how to make
+your PWA installable, how to test to see that it's installable, and try
+doing it yourself.
+
+{% endDetails %}
+
+## Optimal Progressive Web App checklist {: #optimal }
+
+To create a truly great Progressive Web App, one that feels like a
+best-in-class app, you need more than just the core checklist. The optimal
+Progressive Web App checklist is about making your PWA feel native while
+taking advantage of what makes the web powerful.
+
+
+{% Details %}
+{% DetailsSummary 'h3' %}
+
+Provides an offline experience
+
+Where connectivity isn't strictly required, your app works the same offline
+as it does online.
+
+{% endDetailsSummary %}
+
+Where connectivity isn't strictly required, your app works the same offline
+as it does online.
+
+#### Why
+
+In addition to providing a custom offline page, users expect Progressive Web
+Apps to be usable offline. For example, travel and airline apps should have
+trip details and boarding passes easily available when offline. Music,
+video, and podcasting apps should allow for offline playback. Social and
+news apps should cache recent content so users can read it when offline.
+Users also expect to stay authenticated when offline, so design for
+offline authentication. By allowing users to use your PWA while offline,
+you'll create a true app-like experience for them.
```

Comment:
"By allowing users to use your PWA while offline, you'll create a true app-like experience for them."

Change to:

"An offline PWA provides a true app-like experience for users."

Aside from being pithier, 'allowing to use', called an inflated verb, is a writing anti-pattern. If you don't like my suggested rewrite, you need to at least eliminate the inflated verb.